source/reference/api/reverse.txt

```diff
@@ -159,8 +159,7 @@ Embedded Verifier
 ~~~~~~~~~~~~~~~~~
 
 The embedded verifier is enabled by default for replica set
-migrations and performs verification checks for the forward
-direction of reversible sync. When you call the ``/reverse``
+migrations. When you call the ``/reverse``
 endpoint, ``mongosync`` disables the verifier. The verifier
 remains disabled, even after additional calls to the
 ``/reverse`` endpoint.
```

Comment:
Question: Does the `/reverse` endpoint still disable the verifier?

Reply:
It's also mentioned under the `reversible` parameter for [/start](https://deploy-preview-549--docs-cluster-to-cluster-sync.netlify.app/reference/api/start/)

Reply:
@kennethdyer thank you for catching